### DISCUSSION TITLE
Refund pending update error

### DISCUSSION THREAD

**Discussion by justynpride** [HiEventsDev/Hi.Events#111] [Q&A]
I've put in my test Stripe credentials (public, secret and webhook) and can successfully take orders. I've tried refunds and though successful in the app, and in Stripe reports, the app keeps the order status as Refund Pending. 

I'm assuming it should move to something else? I was wondering if I'd missed something with a webhook? I set it up for using the latest API, all the available webhook, and to go to the domain for the system. Any thoughts on what I might have missed? 

If this is best noted as an issue I can post there.

**Comment by daveearley** [comment]
Hi @justynpride 

There are missing details in the documentation about setting up Stripe webhooks in a production environment. You need to set up a webhook to `https://your-domain.com/api/public/webhooks/stripe`. You can do that on [this page](https://dashboard.stripe.com/test/workbench/webhooks) in Stripe.

Once that's set up, you'll be able to see if the webhooks are successful on the Stripe webhook dashboard.

**Reply by justynpride** [reply]
Perfect. My test refunds are going through and updating the order.
